File: 9-palindrome-number/9-palindrome-number.php
class Solution {

    /**
     * @param Integer $x
     * @return Boolean
     */
    function isPalindrome($x) {
		if ($x >= 0) {
			$arr = str_split($x);

			if ($arr == array_reverse($arr)) {
				return true;
			} else {
				return false;
			}
		} else {
			return false;
		}
	}
}

File: 9-palindrome-number/9-palindrome-number.kt
class Solution {
   fun isPalindrome(x: Int): Boolean {
    if (x < 0)
        return false;
    var number = x
    var reverse = 0
    while (number > 0) {
        reverse = reverse * 10 + number % 10
        number /= 10
    }
    return x == reverse
}
}